specs/app_settings/02_user_interface.rst

```diff
@@ -0,0 +1,30 @@
+=====================
+User interface
+=====================
+
+=====  ============  =========  ==========
+epic      title       author     created
+=====  ============  =========  ==========
+`82`_  App settings  tschubotz  2019-01-18
+=====  ============  =========  ==========
+
+.. _82: https://github.com/gnosis/safe/issues/82
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+App settings
+---------------
+
+.. raw:: html
+
+   <img src="screens/android/app_settings.png" width="320px"/>
+
+.. raw:: html
+
+   <img src="screens/ios/app_settings.png" width="320px"/>
+
+* `Zeplin link Android <zpl.io/VYAlyZe>`_
```

Comment:
Missing `https://` thus it's going to be treated as a link from this resource (ie.: `/specs/app_settings/zpl.io/VYAlyZe`)

Reply:
thanks, changed